### ISSUE TITLE
Missing support for variadic functions

### ISSUE THREAD

**Issue by ichekrygin** [timonwong/loggercheck#40]
The current version of the `logrlint` reports a false-positive error/warning on calls with Here’s an arbitrary number of arguments (variadic functions).
```
logger/logger.go:54:28: odd number of arguments passed as key-value pairs for logging (logrlint)
        l.V(DebugLevel).Info(msg, keysAndValues...)
                                  ^
```

**Comment by timonwong** [comment]
The latest logrlint or loggercheck should already addressed this: #8

**Comment by timonwong** [comment]
golangci-lint relases should included latest changes from logrlint/loggercheck in [1.50.0](https://github.com/golangci/golangci-lint/releases/tag/v1.50.0)
